hypershift-operator/controllers/hostedcluster/hostedcluster_webhook_test.go

```diff
@@ -1326,3 +1329,157 @@ func TestValidateKVNodePoolUpdate(t *testing.T) {
 		})
 	}
 }
+
+// util function used to generate a service map that is different than the defaults
+func customKubeVirtServiceMap() []v1beta1.ServicePublishingStrategyMapping {
+	// use the defaults as a basis
+	defaults := apiexample.GetIngressServicePublishingStrategyMapping(v1beta1.OVNKubernetes, false)
+
+	custom := []v1beta1.ServicePublishingStrategyMapping{}
+	for _, cur := range defaults {
+		entry := v1beta1.ServicePublishingStrategyMapping{
+			Service: cur.Service,
+		}
+		// none of the kubevirt defaults use nodeport, so this
+		// is an easy way to make a service map different than
+		// the default
+		entry.ServicePublishingStrategy.NodePort = &v1beta1.NodePortPublishingStrategy{}
+		custom = append(custom, entry)
+	}
```

Comment:
Please use `make` to create an empty slice. Here we even know the required length so we don't need to use append. e.g.
```suggestion
	custom := make([]v1beta1.ServicePublishingStrategyMapping, len(defaults))
	i := 0
	for _, cur := range defaults {
		entry := v1beta1.ServicePublishingStrategyMapping{
			Service: cur.Service,
		}
		// none of the kubevirt defaults use nodeport, so this
		// is an easy way to make a service map different than
		// the default
		entry.ServicePublishingStrategy.NodePort = &v1beta1.NodePortPublishingStrategy{}
		custom[i] = entry
		i++
	}
```